mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -83,6 +217,9 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 		FIELD field_23184 blockVisionPredicate Lnet/minecraft/class_4970$class_4973;
 		FIELD field_23185 postProcessPredicate Lnet/minecraft/class_4970$class_4973;
 		FIELD field_23186 inWallOverlayPredicate Lnet/minecraft/class_4970$class_4973;
+		METHOD <init> (Lnet/minecraft/class_3614;Ljava/util/function/Function;)V
+			ARG 1 material
+			ARG 2 materialColor
```

Comment:
Hmm, is this a `materialColorFactory`

Reply:
That's a good point. I think we should name the field/arg `materialColorFactory` or `colorFactory` and have the method still be named `materialColor`